hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -394,6 +394,7 @@
             '''
           resval: "1" },
         { bits: "5", name: "csb",        desc: "Direct input of CSb signal", resval: "1" },
+        { bits: "6", name: "tpm_csb",    desc: "Direct input of TPM CSb", resval: "1" }
```

Comment:
We'll want to get this added to the DIFs, too.

Reply:
could you let me know how? I vaguely remember there's a generation script but cannot find it.. :(